CONTRIBUTING.md

```diff
@@ -184,6 +184,256 @@ Here are some dependents with specific version:
 
 When you develop the Sealer project at the local environment, you should use subcommands of Makefile to help yourself to check and build the latest version of Sealer. For the convenience of developers, we use the docker to build Sealer. It can reduce problems of the developing environment.
 
+### Run E2E Test or Write E2E Test Cases
+Before you commit a pull request, you need to run E2E test first. If you have modified the code or added features, you need to write corresponding E2E test cases, or you want to contribute E2E test cases for sealer, please read this section.
+
+#### Principle of sealer k8s in docker
+
+Sealer E2E test is inspired by kind, uses a container to emulate a node by running systemd inside the container and hosting other processes with systemd.
+Sealer uses（sealer/pkg/infra/container/imagecontext/base/Dockerfile）to build a basic image: `sealerio/sealer-base-image:v1`，and use this image to start the container to emulate node。
```

Comment:
Cloud you please add a link for sealer/pkg/infra/container/imagecontext/base/Dockerfile

Reply:
OK